css/enhancedsteam.css

```diff
@@ -1685,6 +1685,13 @@ input[type=checkbox].es_dlc_selection:checked + label {
 	font-size: 12px;
 }
 
+/***************************************
+ * Regional Price Bugfix
+ **************************************/
+ /* .ds_options_tooltip {
+	 z-index: 99999 !important;
+ } */
+
```

Comment:
What should this be?

Reply:
https://github.com/tfedor/AugmentedSteam/issues/389

Reply:
Wait, why is this commented out?